react-accordion-component/my-app/src/Accordion.js

```diff
@@ -0,0 +1,42 @@
+import { useState } from 'react';
+import './Accordion.css';
+
+export default function Accordion({ languages }) {
+  const [clicked, setClicked] = useState('');
```

Comment:
`clicked` is ambiguous. I think you are using this to decide if the language should be opened or closed, so `isOpen` would be better.